src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: $gap;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: $small-screen) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: $middle-screen) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: $large-screen) {
+    grid-template-columns: repeat(4, $card-width);
+  }
 
   &__card {
-    display: flex;
+    display: grid;
```

Comment:
You don't need grid for card

Reply:
> You don't need grid for card
I apologize. It is not specified here
![image](https://user-images.githubusercontent.com/103377396/174784775-f440c354-6f67-4e2d-996a-f4ac58d332cf.png)